cgi-bin/LJ/Web.pm

```diff
@@ -2910,8 +2879,8 @@ sub res_includes {
             cmax_comment        => LJ::CMAX_COMMENT,
         );
 
-        my $site_params     = LJ::js_dumper( \%site );
-        my $site_param_keys = LJ::js_dumper( [ keys %site ] );
+        my $site_params     = to_json( \%site );
+        my $site_param_keys = to_json( [ keys %site ] );
```

Comment:
Looks like site_param_keys is no longer needed due to another change below that removed the one place it was used.

Reply:
Oh good catch!